src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -17680,8 +17690,14 @@ public class Cls
 {
     public static void Main()
     {
-        var list = new Dictionary<int, long> { [out var x1] = 3 };
-        System.Console.WriteLine(x1);
+        var list = new Dictionary<int, long>
+        {
+            [out var x1] = 3,
+            [out var _] = 4,
+            [out _] = 5
```

Comment:
The lack of warning on `out` here is because there is already an error on the identifier. See `BindElementOrIndexerAccess` which does this shortcut.